ros-test_franka_simulation_bringup/install.bash

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# shellcheck disable=SC1090
+
```

Comment:
I would say Coppelia should be in a separate target.

What do you think @juandhv ?